logging/logger.go

```diff
@@ -72,11 +72,16 @@ func (l *Logger) AddTagToSession(sessionId, key, value string) {
 	addTag(l.writer, EntitySession, sessionId, key, value)
 }
 
-// SessionEnd marks the specified session as ended
+// SessionEnd marks the specified session as ended (/*deprecated please use EndSession instead*/)
```

Comment:
https://go.dev/wiki/Deprecated

Reply:
so I should remove the deprecation message?

Reply:
it says this

If function F1 is being replaced by function F2 and the first release in which F2 is available is Go 1.N, then an official deprecation notice for F1 should not be added until Go 1.N+1. This ensures that Go developers only see F1 as deprecated when all supported Go versions include F2 and they can easily switch.

Reply:
No no - I meant the links mentioned in the above page.

Here is one reference they have from the Go std library - https://cs.opensource.google/search?q=Deprecated:%20language:go&ss=go%2Fgo

Reply:
acha okay